doc/tutorials/imgproc/hitOrMiss/hitOrMiss.markdown

```diff
@@ -43,11 +44,27 @@ You can see that the pattern is found in just one location within the image.
 Code
 ----
 
-The code corresponding to the previous example is shown below. You can also download it from
-[here](https://github.com/opencv/opencv/tree/master/samples/cpp/tutorial_code/ImgProc/HitMiss.cpp)
-@include samples/cpp/tutorial_code/ImgProc/HitMiss.cpp
+The code corresponding to the previous example is shown below.
+
+@add_toggle_cpp
+You can also download it from
+[here](https://github.com/opencv/opencv/tree/master/samples/cpp/tutorial_code/ImgProc/HitMiss/HitMiss.cpp)
```

Comment:
Please use "raw" URLs for files you embed to not overload the main server:
https://raw.githubusercontent.com/opencv/opencv/master/samples/cpp/tutorial_code/ImgProc/HitMiss.cpp